### PR TITLE
Change "Disable Filter" to "Enable Filter" on Font Import window

### LINKED ISSUE #6154: Tricky name on Font Import option
Hi.

In the Font Import Panel there is an option named "Disable Filter" that has to be 'On' if we want to import the bitmap font without filter and 'Off' if we want it with filter. Wouldn't be easier to just name the option "Filter" or "Use Filter" (as in DynamicFonts) an have it 'On' or 'Off' if we want to use filter or not? What you guys think?

font_import

### PR DESCRIPTION
**Pull request by volzhs**
Fix #6154

### CODE REVIEW

**Comment by neikeq** [comment]
Hmm, wouldn't this break compatibility? Should we wait for 3.0?

**Comment by volzhs** [comment]
As what I tested, It's just related with importing font process.
I don't think it will break.

**Comment by bojidar-bg** [comment]
@volzhs Maybe check for the old property, and if it is present (and the new one isn't), use it instead?

**Comment by volzhs** [comment]
@bojidar-bg good point. now I need to investigate how auto-reimport does.

**Comment by volzhs** [comment]
Tested with `.fnt` that has `disable_filter` option and works as expected. :)

File: tools/editor/io_plugins/editor_font_import_plugin.cpp [hunk 1]
```diff
@@ -1592,6 +1592,9 @@ Ref<BitmapFont> EditorFontImportPlugin::generate_font(const Ref<ResourceImportMe
 	int top_space = from->get_option("extra_space/top");
 	int bottom_space = from->get_option("extra_space/bottom");
 	bool enable_filter = from->get_option("advanced/enable_filter");
+	if (from->has_option("advanced/disable_filter")){
```
Comment (akien-mga) [inline tools/editor/io_plugins/editor_font_import_plugin.cpp]: Could you add a comment saying that this is a compatibility check for a deprecated option? This way we could drop it for 3.0 or later if we want.

**Comment by akien-mga** [comment]
Fine to merge IMO.

File: tools/editor/io_plugins/editor_font_import_plugin.cpp (hunk 1 above)
Comment (volzhs) [inline tools/editor/io_plugins/editor_font_import_plugin.cpp]: it's done.

**Pull request merged**